packages/paste-core/components/form-pill-group/src/PillCloseIcon.tsx

```diff
@@ -57,7 +56,6 @@ export const PillCloseIcon: React.FC<PillCloseIconProps> = ({
       borderStyle="solid"
       borderColor="transparent"
       transition="color 150ms ease-in, border-color 150ms ease-in, background-color 150ms ease-in"
-      {...computedStyles}
```

Comment:
Question: is there any chance these computed styles are intentionally set after the default style props? Does the computation calculate a state that sometimes needs to override a style prop that is fine _most of the time_, but not always?

Reply:
Should get caught by VRT but I know we don't VRT hover and focus styles for example

Reply:
Had to move those to address TS errors since importing instead of requiring deepmerge resulted in actual typing instead of `any`. Will see if it causes visual diffs otherwise we could ts-ignore if needed.

Reply:
Verified hover/focus styles are not impacted by this change.